src/commands/fuelup.rs

```diff
@@ -0,0 +1,81 @@
+use anyhow::{bail, Result};
+use clap::Parser;
+use tracing::info;
+
+use crate::{
+    constants::{FUELUP_RELEASE_DOWNLOAD_URL, GITHUB_API_REPOS_BASE_URL, RELEASES_LATEST},
+    download::{download_file_and_unpack, fuelup_bin_dir, get_latest_tag, unpack_extracted_bins},
+};
+
+#[derive(Debug, Parser)]
+pub enum FuelupSubcommand {
```

Comment:
```suggestion
pub enum FuelupCommand {
```
^ to be consistent with `InstallCommand` maybe?

Reply:
Hmmm, I was thinking about the naming of this for a bit. What I wanted to achieve was to separate a `Command` without additional `Command`s but now that I look at it again it does seem wrong that it is called a `Subcommand`... I think perhaps I'll indeed change it back so that it's consistent with `forc`